app/workers/event_worker.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class EventWorker
```

Comment:
This shouldn't be in this PR. This pr should be all about user specs. Then have another PR about sidekiq/events. I won't make you change this but in the future try to keep your PR's more focused on a single thing. "Update User specs", "Create events functionality with sidekiq", etc.